paper_sections/05-computational_and_statistical_issues.Rmd

```diff
@@ -628,9 +636,22 @@ to balance penalties between nested smooth terms (this is why nested random
 
-[^gsnull]: For model *GS* both the factor-smoother, and tensor products of random effect ("re") and other smooth terms do not have a penalized nullspace by construction (they are full rank), as noted above. For model *GI* groupwise terms, we used basis types that had a penalty added to the nullspace, so called "shrinkage" methods: `bs="ts"`, `"cs"`, or `"ps"` have this property.
+As noted above, interpreting the shape of global terms and group-wise deviations seperately for  *GS* models fit using tensor-product group-level terms is complicated by the fact that **mgcv** will drop some basis-functions from the group-level terms to prevent perfect concurvity. 
+For tensor-product smoothers, **mgcv** will generally drop $\le k$ terms from the group-level smoother, where $k$ is the number of basis functions in the global smoother.
+The total number of terms dropped will depend on the smoothers used for the global and group-level terms, and will be dropped essentially at random from different grouping levels. 
+This means that some groups will have a different range of potential deviations from the global smoother than others.
+This has the effect of also somewhat altering the shape of the global smooth relative to what it would be based on model *G* (the average curve through all the data); this will be a larger issue when the number of basis functions in the global smooth and the number of group levels are small. 
+We have tested the effect of this issue on our simulated `bird_move` data set and did not find that it lead to substantial bias in estimating the shape of the global smoother, relative to the amount of bias inherent in any smooth estimation method[^global_note]. As noted in section III, we found that `t2()` tensor product smoothers with full penalties (`full = TRUE` in the `t2` function) for group-level smoothers showed the best performance at recreating the true global function from our simulated `bird_move` data set, compared to other possible types of tensor product.
+However, more work on when these models accurately reconstruct global effects is still needed.
```

Comment:
this paragraph is a good compromise here. It might be worth slipping in something about how we're not really modelling the "true" global smooth, but rather an "Average" smooth (you kind of hint at this. The important difference being (I think) if you measured fishos, but you had sampling bias towards big fishos, the "true" global fisho-env relationship isn't captured, just deviations from average big fisho... (apologies for all the scare quotes)

Reply:
It might be the late hour, but I'm actively fighting the urge to put "fisho" in the text. Also, I can take a stab at adding a bit of text on that.